### PR TITLE
[SDL 0109] SetAudioStreamingIndicator RPC

### PR DESCRIPTION
**Pull request by ghost**
This PR implements #67 according to the SDL proposal 0109

This PR is **ready** for review.

### Risk
This PR makes **minor** API changes.,

### CLA
- [X] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: MOBILE_API.xml [hunk 1]
```diff
@@ -891,6 +891,27 @@
...
         </element>
     </enum>
+	
+    <enum name="AudioStreamingIndicator" since="5.0">
+        <element name="PLAY_PAUSE">
+            <description>
+                Default playback indicator.
+                By default the playback indicator should be PLAY_PAUSE when:
+                    - the media app is newly registered on the head unit (after RegisterAppInterface)
+                    - the media app was closed by the user (App enteres HMI_NONE)
```
Comment (theresalech) [inline MOBILE_API.xml]: This should say "enters `HMI_NONE" (removing extra "e" and adding code blocks around "HMI_NONE")

File: MOBILE_API.xml [hunk 2]
```diff
@@ -891,6 +891,27 @@
             <description>Similar to VP8, but VP9 is customized for video resolutions beyond high-definition video (UHD) and also enables lossless compression.</description>
         </element>
     </enum>
+	
+    <enum name="AudioStreamingIndicator" since="5.0">
+        <element name="PLAY_PAUSE">
+            <description>
+                Default playback indicator.
+                By default the playback indicator should be PLAY_PAUSE when:
+                    - the media app is newly registered on the head unit (after RegisterAppInterface)
```
Comment (theresalech) [inline MOBILE_API.xml:900]: When comparing to the [proposal](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0109-set-audio-streaming-indicator.md), it seems "RegisterAppInterface" should be in a code block

File: MOBILE_API.xml [hunk 3]
```diff
@@ -891,6 +891,27 @@
...
+                    - the media app was closed by the user (App enteres HMI_NONE)
+                    - the app sends SetMediaClockTimer with audioStreamingIndicator not set to any value
```
Comment (theresalech) [inline MOBILE_API.xml:902]: When comparing to the [proposal](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0109-set-audio-streaming-indicator.md), it seems "SetMediaClockTimer" and "audioStreamingIndicator" should be in code blocks.

**Review by theresalech — CHANGES_REQUESTED** [review]
@kshala-ford I've left a couple of comments.  Can you please review and address accordingly?  Let me know of any questions.  Thanks!

File: MOBILE_API.xml (hunk 2 above)
Comment (jacobkeeler) [reply MOBILE_API.xml:900]: Code blocks are not used in the spec

File: README.md [hunk 4]
```diff
@@ -613,6 +613,17 @@ Enum for each type of video streaming codec.
 |`VP9`|Similar to VP8, but VP9 is customized for video resolutions beyond high-definition video (UHD) and also enables lossless compression.|
 
 
+### AudioStreamingIndicator
+##### Elements
+
+| Value | Description | 
+| ---------- |:-----------:|
+|`PLAY_PAUSE`|Default playback indicator.                By default the playback indicator should be PLAY_PAUSE when:                    - the media app is newly registered on the head unit (after RegisterAppInterface)                    - the media app was closed by the user (App enteres HMI_NONE)                    - the app sends SetMediaClockTimer with audioStreamingIndicator not set to any value            |
```
Comment (theresalech) [inline README.md]: Please fix typo "enteres"

**Review by theresalech — CHANGES_REQUESTED** [review]
@jacobkeeler can you please fix the typo in the readme?

File: README.md (hunk 4 above)
Comment (jacobkeeler) [reply README.md]: Fixed with https://github.com/smartdevicelink/rpc_spec/pull/108/commits/3e9a1a49b7c54635a7c73cfbec38b7548e9e9e03

**Review by theresalech — APPROVED** [review]

**Review by joeygrover — APPROVED** [review]

**Pull request merged**